addons/io/org.openhab.io.neeo/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<artifactId>pom</artifactId>
+		<groupId>org.openhab.io</groupId>
+		<version>2.4.0-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>org.openhab.io.neeo</artifactId>
+	<packaging>eclipse-plugin</packaging>
+
+	<name>openHAB NEEO Transport</name>
+
+	<dependencies>
```

Comment:
Do you really need to declare these dependencies here? The package imports in the manifest should be enough forthe resolution through Tycho afaik.